sensor/common/sensor/sensor.go

```diff
@@ -217,8 +217,9 @@ func (s *Sensor) Start() {
 		s.AddNotifiable(wrapNotifiable(koCacheSource, "Kernel object cache"))
 	}
 
-	// Enable endpoint to retrieve vulnerability definitions if local image scanning is enabled.
-	if env.LocalImageScanningEnabled.BooleanSetting() {
+	// Enable endpoint to retrieve vulnerability definitions if local image scanning or Scanner v4 is enabled.
+	// Scanner v4 / Node Indexing requires access to the repo to cpe mapping file hosted by central.
+	if env.LocalImageScanningEnabled.BooleanSetting() || env.NodeIndexEnabled.BooleanSetting() {
```

Comment:
Could we make sure that if `LocalImageScanningEnabled` is disabled, then this endpoint is still serving the mapping? Have you tested that manually maybe?

Reply:
Yes, I've tested that manually. In fact, local image scanning is disabled by default, so by default the route is not served.
I tested manually, by enabling either of the vars in Sensor and verifying that the route is served.